charts/spire/charts/spire-server/templates/roles.yaml

```diff
@@ -12,6 +12,15 @@ rules:
     verbs: [get]
     # allow access to "get" and "patch" the spire-bundle ConfigMap (for SPIRE
     # agent bootstrapping, see the spire-bundle ConfigMap below)
+---
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{ include "spire-server.fullname" . }}-bundle
+  namespace: {{ .Values.notifier.k8sbundle.namespace | default .Release.Namespace }}
+rules:
+    # allow access to "get" and "patch" the spire-bundle ConfigMap (for SPIRE
+    # agent bootstrapping, see the spire-bundle ConfigMap below)
```

Comment:
The previous role/rolebinding is split into two because the role/rolebinding only provides permissions into the namespace the role/rolebinding is created in. In this case, the ability to deal with the configmap is in the other namespace so needs separate objects.